truststore_java.go

```diff
@@ -110,7 +110,13 @@ func (m *mkcert) execKeytool(cmd *exec.Cmd) ([]byte, error) {
 	out, err := cmd.CombinedOutput()
 	if err != nil && bytes.Contains(out, []byte("java.io.FileNotFoundException")) {
```

Comment:
This entire block should be skipped on Windows, as it's pointless to retry if we are not adding sudo.

```suggestion
	if err != nil && bytes.Contains(out, []byte("java.io.FileNotFoundException")) && runtime.GOOS != "windows" {
```